src/theme.ts

```diff
@@ -0,0 +1,25 @@
+import { createStitches, createTheme } from "@stitches/react";
+
+export const { styled, css } = createStitches({
+  theme: {
+    colors: {
+      text: "black",
+      focus: "rgba(3, 138, 255, 0.1)",
+      border: "rgba(0, 0, 0, 0.1)",
+      bg1: "white",
+      bg2: "rgba(0, 0, 0, 0.02)",
+      btnText: "rgba(0, 0, 0, 0.6)"
+    },
+  },
+});
+
+export const darkTheme = createTheme({
+  colors: {
+    text: "white",
+    focus: "rgba(3, 138, 255, 0.4)",
+    bg1: "#18181B",
+    bg2: "rgba(255, 255, 255, 0.1)",
+    border: "rgba(255, 255, 255, 0.1)",
+    btnText: "rgba(255, 255, 255, 0.6)",
```

Comment:
while this is fine, I have a few nitpicks:
- I think it's better to name b1 and bg2 differently, cuz it's not clear what that means (the other ones, I can tell mostly)
- ultra minor, but lets pick the same color format for all the colors (just for consistency!). Can be either hex or rgba, but not both (plus color names) haha

Reply:
Ok, how about now?